src/re_complete/app.cljs

```diff
@@ -151,12 +151,13 @@
           (clear-selected-item linked-component-key))))
 
 (defn keys-handling [linked-component-key onclick-callback]
-  (events/listen js/window "keydown"
-                 (fn [e]
-                   (let [key-code (.-keyCode e)]
-                     (when (#{13 38 40 9} key-code)
-                       (.preventDefault e)
-                       (if (or (= key-code 13) (= key-code 9))
-                         (do (dispatch [:keys-handling linked-component-key key-code])
-                             (when onclick-callback (onclick-callback)))
-                         (dispatch [:keys-handling linked-component-key key-code])))))))
+  (.addEventListener js/window "keydown"
```

Comment:
Shall we change this? I would say so, but If not, let's remove `goog.events` if not needed.